### ISSUE TITLE
Button: documentatie website

### ISSUE THREAD

**Issue by eslook** [nl-design-system/documentatie#3200]
In Stappenplan Canvas https://codefornl.slack.com/docs/T68FXPFQV/F09BCCLGKHB?focus_section_id=temp:C:ccN3fb3665efb7e4492a1f5d8dab

**Accessibility en inclusie documentatie beschikbaar voor de overwegingen die van toepassing zijn.**

*  Documenteer toegankelijkheidseisen en -overwegingen:
    *  Verwijs naar relevante WCAG richtlijnen
    *  Leg uit hoe de component hieraan voldoet
    *  Documenteer eventuele uitzonderingen of beperkingen
    * Publiceer op de acceptatiecriteria voor gebruik op de website
    * Zet de acceptatiecriteria voor de component klaar
    *  Geef voorbeelden van inclusief gebruik
*  Voeg voorbeelden en code samples toe:
    *  Toon correcte implementatie voor toegankelijkheid
        * Het mooist zou zijn als we dit uit de stories kunnen gebruiken
        * Op dit moment zijn het handmatig bijgehouden code snippets op de website
        *  Documenteer best practices als ze er zijn (semantiek, linkjes naar richtlijnen, meninge van slimme andere mensen, etc.)

**Gebruikt component API’s gedocumenteerd**

* Gebruik van component is gedocumenteerd op de website.
* Storybook hergebruikt de documentatie van het gebruik van de component in code en algemeen gebruik van de component naar de website.
* Figma verwijst voor de documentatie over het gebruik van de component naar de documentatie op de website.
* Figma specifieke documentatie over de component staat in Figma (voor nu).
* Component wordt gerendered als voorbeeld op de website, met dezelfde configuratie(?) als de default component in Storybook en Figma
* Design tokens worden gepubliceerd in een design tokens npm package

**Acceptatiecriteria**

* Gebruik van candidate React component is gedocumenteerd in {component}-react/README.md 
* Gebruik van candidate CSS component is gedocumenteerd in {component}-css/README.md 
* {component}-react/README.md en {component}-css/README.md worden gebruikt op nldesignsystem.nl/{component}

**Comment by eslook** [comment]
Blocked by most issues.
